### PR TITLE
feat: add miracl as retrieval task

### PR DESCRIPTION
**Pull request by violenil**
| model | ndcg@10 | evaluation_time (seconds) |
| --- | --- | --- |
| multilingual-e5-base | 80.22 % | 17.13 |
| multilingual-e5-large | 82.01 % | 52.19 |

### CODE REVIEW

**Review by guenthermi — APPROVED** [review]
LGTM, but we need to figure out how to handle the languages first. Maybe we keep it mono-linugal. Then it might make sense to add ES or something like this to the name for the case that we prepare MIRACL for other languages as well.

**Comment by violenil** [comment]
> we need to figure out how to handle the languages first
Totally agree

File: mteb/tasks/Retrieval/MIRACLRetrieval.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
+from datasets import load_dataset
+
+from ...abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+_EVAL_SPLIT = 'test'
+
+
+class MIRACLRetrieval(AbsTaskRetrieval):
```
Comment (guenthermi) [inline mteb/tasks/Retrieval/MIRACLRetrieval.py]:
```suggestion
class MIRACLESRetrieval(AbsTaskRetrieval):
```

**Comment by Markus28** [comment]
@guenthermi Regarding the multilinguality: The abstract `AbsTaskRetrieval` class in MTEB currently doesn't support multilingual retrieval datasets, but I did extend it in the long-douments fork to do that: https://github.com/jina-ai/mteb-long-documents/blob/0b6b00eca96b5d9f62a80eb35d7176a99e91306e/mteb/abstasks/AbsTaskRetrieval.py

**Comment by guenthermi** [comment]
> @guenthermi Regarding the multilinguality: The abstract `AbsTaskRetrieval` class in MTEB currently doesn't support multilingual retrieval datasets, but I did extend it in the long-douments fork to do that: https://github.com/jina-ai/mteb-long-documents/blob/0b6b00eca96b5d9f62a80eb35d7176a99e91306e/mteb/abstasks/AbsTaskRetrieval.py

ok, but is it then merged into the global MTEB? I thought we merged the changes from the long-context fork.

**Comment by Markus28** [comment]
No it's not merged. I added it for the code tasks, which have neither been merged into MTEB nor into the master branch of the long-documents fork.

**Pull request merged**